badge-maker/lib/make-badge.js

```diff
@@ -4,6 +4,7 @@ const { normalizeColor, toSvgColor } = require('./color')
 const badgeRenderers = require('./badge-renderers')
 const { stripXmlWhitespace } = require('./xml')
 const { DEFAULT_LOGO_HEIGHT } = require('./constants')
+const { getIconSize } = require('./svg-helpers.js')
```

Comment:
`svg-helpers.js` resides at `/lib` and not inside the lib of badge-maker `/badge-maker/lib`
:dizzy_face: